cairo/ethereum/utils/numeric.cairo

```diff
@@ -535,6 +536,17 @@ func U384_to_be_bytes{range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(
     return result;
 }
 
+func U384_to_le_bytes{range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(
+    value: U384, length: felt
+) -> Bytes {
+    alloc_locals;
+    let bytes_be = U384_to_be_bytes(value, length);
+    let (local bytes_result: felt*) = alloc();
+    reverse(bytes_result, length, bytes_be.value.data);
```

Comment:
cant we avoid a reverse here?

Reply:
As it leverages `U384_to_be_bytes` the reverse is necessary. 
It would be possible to duplicate most of the code from `U384_to_be_bytes` and adapt it for le though.

Reply:
> It would be possible to duplicate most of the code from U384_to_be_bytes and adapt it for le though.

That's preferable

Reply:
It's not really duplicating as you'd be iterating over the limbs in the reversed order directly. Reversing the array here is super inefficient when we can directly work in the correct endianess

Reply:
I was wrong on this indeed. Made the change with [0ebf1d6](https://github.com/kkrt-labs/keth/pull/1335/commits/0ebf1d69e7d86734b70ff537c7b5fb47c8e8d1cf)